backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/InteropLowering.kt

```diff
@@ -674,6 +674,36 @@ internal class InteropLoweringPart1(val context: Context) : IrBuildingTransforme
         }
     }
 
+    override fun visitProperty(declaration: IrProperty): IrStatement {
+        val backingField = declaration.backingField
+        return if (declaration.isConst && backingField?.isStatic == true && context.config.isInteropStubs) {
```

Comment:
Maybe comment why this is needed?

Reply:
What do you mean by "this"? The entire transformation or the check above?

Reply:
The whole transformation.

Reply:
It is just an optimization and shouldn't be required by our memory model. Does it still need any comments?

Reply:
It better should be commented anyway, as this is pretty nontrivial transformation.

Reply:
Done?

Reply:
Yep